w3a2.c

```diff
@@ -0,0 +1,40 @@
+#include<stdio.h>
+#include<stdlib.h>
+
+    main()
+     { 
+	  int arr[10],a,p,i,j,temp;
+  
+       printf("enter array size --> ");
+       scanf("%d",&a);
+       
+       printf("enter array elements --> ");
+      
+	  for(i=0;i<a;i++) 
+	   {
+	      scanf("%d",&arr[i]);
+	   }
+	   
+       printf("enter kth smallest element to be searched -->\n");
+       scanf("%d",&p);
+		   
+      for(i=1;i<a;i++)

```

Comment:
2) if the number of elements in the array are less 10 or 20 sorting is preferable to find kth smallest number because we won't have to go through each element and compare with one another multiple times,  but if size of the array is huge then it will create some issues as time complexity of the program will increase